packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -341,6 +345,11 @@ export const Combobox = <T extends string | number>({
             })}
           </ul>
         )}
+        {asyncError && (
+          <Stack justifyContent="center" alignItems="center" height={8}>
+            <Text color="secondary">{t('combobox.async.error', 'An error occurred while loading options.')}</Text>
```

Comment:
Could you maybe add a warning triangle icon here? Either next to the text or above.